pod/perlsyn.pod

```diff
@@ -549,6 +558,24 @@ followed by C<my>.  To use this form, you must enable the C<refaliasing>
 feature via C<use feature>.  (See L<feature>.  See also L<perlref/Assigning
 to References>.)
 
+As of Perl 5.36, you can iterate over a list of lexical scalars n-at-a-time.
```

Comment:
I think this wording `list of lexical scalars` should also be changed here (like it was elsewhere) to `iterate over multiple values` or something. Maybe be consistent with all of them?

Reply:
Yes, consistency is important. I fail!

I changed it to this text - can it be improved further, or are we good enough?

```
As of Perl 5.36, you can iterate over multiple values at a time.
You can only iterate with lexical scalars as the iterator variables - unlike
list assignment, it's not possible to use C<undef> to signify a value that
isn't wanted.  This is a limitation of the current implementation, and might
be changed in the future.
```

Reply:
Wfm, thanks